src/sentry/_importchecker.py

```diff
@@ -117,10 +131,18 @@ def checking_import(name, globals=None, locals=None, fromlist=(), level=0):
         return real_import(name, globals, locals, fromlist, level)
 
     to_name = resolve_full_name(package, name, level)
+    now = time.time()
```

Comment:
this [should use time.monotonic or time.perf_counter](https://youtu.be/hb-12mgQrzM)